src/phoenix/db/models.py

```diff
@@ -156,6 +156,24 @@ class Project(Base):
     )
 
 
+class TraceSession(Base):
+    __tablename__ = "trace_sessions"
+    id: Mapped[int] = mapped_column(primary_key=True)
+    session_id: Mapped[str] = mapped_column(String, nullable=False, unique=True)
```

Comment:
Do we need an index on this column?

Reply:
you're right. i missed it. thx

Reply:
oh wait. `unique` already has index

Reply:
Interesting, did not know that.

Reply:
fyi, here are the query plans for sqlite and postgres

### sqlite

<img width="665" alt="Screenshot 2024-10-15 at 11 41 58 AM" src="https://github.com/user-attachments/assets/d485c236-a2bf-4160-b0b8-b94b96bee81d">

### postgresql

<img width="835" alt="Screenshot 2024-10-15 at 11 44 45 AM" src="https://github.com/user-attachments/assets/6dfe1b6e-c0f1-4d6f-b09f-094f8cb54b6d">